tests/unit/dependabot_alerts/cli_test.py

```diff
@@ -1,23 +1,10 @@
-from importlib.metadata import version
```

Comment:
I think this may be an issue with the cookiecutter: puts this unused import here. Not sure why that's not causing the cookiecutter's tests to fail.